organizations/models.py

```diff
@@ -28,6 +34,10 @@ class Organization(TimeStampedModel):
     def __unicode__(self):
         return u"{name} ({short_name})".format(name=self.name, short_name=self.short_name)
 
+    def clean(self):
+        if not re.match("^[a-zA-Z0-9_-]*$", self.short_name):
```

Comment:
Where does this regex come from? Can regex from the xblock library be used directly?

Reply:
Same regex is being used in Publisher. I just copied from there. 
Here is the link:
https://github.com/edx/course-discovery/pull/1203/files

Reply:
Where did publisher get its regex? The thing I worry about is there not being a canonical check for what characters can or cannot be used in the organization part of a course id. What happens if the regex in publisher changes? Would this check have to change as well? How does studio currently prevent bad orgs or course ids? Is it the same or slightly different?

I'm going to give a thumbs up to this PR because I think we can start seeing value from it right away, but I think these are questions we will probably want to answer.

Reply:
Studio prevents bad orgs like this:
https://github.com/edx/edx-platform/blob/163ff8469994df8aca24c11db4d5a74c0b7fc3bf/common/static/common/js/components/utils/view_utils.js#L212-L231
The thing is all(studio, django admin, publisher) allow same characters.

Reply:
ah

Reply:
So the regex is spread out across javascript and python. Fun :)

Reply:
Yes!

Reply:
> So the regex is spread out across javascript and python. Fun

I wish there was a "terrified, of-course-thats'-how-it-works eyes" emoji I could react with here